pkgs/development/python-modules/pyu2f/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, fetchPypi, buildPythonPackage,
+  six, mock, pyfakefs, unittest2
+}:
+
+buildPythonPackage rec {
+  pname = "pyu2f";
+  version = "0.1.4";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "12d1260ccd7cfd7a987fc1ad84667770efd6b422cec47fe8eda18d5d52ff6bd3";
+  };
+
+  propagatedBuildInputs = [ six ];
+
+  buildInputs = [ six mock pyfakefs unittest2 ];
```

Comment:
These appear to only be to used for tests https://github.com/google/pyu2f/blob/8742d798027a21cbde0704aac0e93269bad2c3d0/setup.py#L42 so they belong in `checkInputs`.

Things in `checkInputs` actually get added to `buildInputs` but only when tests are enabled.
Which they are enabled by default in nixpkgs for python.